supabase_auth/types.py

```diff
@@ -19,48 +25,56 @@
 
 from typing_extensions import Literal, NotRequired, TypedDict
 
-Provider = Literal[
-    "apple",
-    "azure",
-    "bitbucket",
-    "discord",
-    "facebook",
-    "figma",
-    "fly",
-    "github",
-    "gitlab",
-    "google",
-    "kakao",
-    "keycloak",
-    "linkedin",
-    "linkedin_oidc",
-    "notion",
-    "slack",
-    "slack_oidc",
-    "spotify",
-    "twitch",
-    "twitter",
-    "workos",
-    "zoom",
```

Comment:
`zoom` is missing from the Provider enum, is that intended?

Reply:
Very good catch!, fixed.